web/src/compose_actions.js

```diff
@@ -128,7 +128,7 @@ export function complete_starting_tasks(msg_type, opts) {
 export function maybe_scroll_up_selected_message() {
     // If the compose box is obscuring the currently selected message,
     // scroll up until the message is no longer occluded.
-    if (message_lists.current.selected_id() === -1) {
+    if (message_lists.current?.selected_id() === -1) {
         // If there's no selected message, there's no need to
         // scroll the compose box to avoid it.
         return;
```

Comment:
I think for this placement, we want to just assert `message_lists.current` at the top of the function, or bail early if it is undefined.

Reply:
Changed to returned early `message_lists.current` is undefiend.